### ISSUE TITLE
[Feature] Add keyboard & mouse shortcut list to user docs or app

### ISSUE THREAD

**Issue by j-o-e-s** [manga-download/hakuneko#6847]
**Is your feature request related to a problem? Please describe.**
Is it possible to add a full list of keyboard shortcuts to the user documentation and/or the app? Its been difficult to hunt down what can be done just by hovering over icons in the app.  Also, the double-click chapter to open its downloaded folder is not documented, or I haven't found it.

**Describe the solution you'd like**
An entry in the user documentation with all keyboard shortcuts and mouse clicks. Or, a pop-up window with all this listed.

**Describe alternatives you've considered**
Writing it all down myself from various forums and user comments.

**Context (please complete the following information):**
 - Your current HakuNeko Version 6.1.7

**Comment by Sheepux** [comment]
Won't be done in haku, but i'll do something in haru

**Comment by MikeZeDev** [comment]
Should i close it them?

**Comment by Sheepux** [comment]
No, haru will probably be merged back to haku once it's stable. Such Issue will still be open and must stay here. 
That's why i'm not closing them myself when I comment
this is why they're all referenced here: https://github.com/manga-download/haruneko/issues/40
